lib/resources/collection/index.js

```diff
@@ -448,7 +448,7 @@ Collection.prototype.save = function (ctx, fn) {
 
   domain.changed =  function (property) {
```

Comment:
That would be nice I guess, it's a feature that doesn't break anything. I can update the PR with it if you want.